pyvista/_vtk.py

```diff
@@ -263,7 +263,11 @@
         vtkTrimmedExtrusionFilter,
     )
     from vtkmodules.vtkFiltersParallel import vtkIntegrateAttributes
-    from vtkmodules.vtkFiltersParallelDIY2 import vtkRedistributeDataSetFilter
+
+    try:
+        from vtkmodules.vtkFiltersParallelDIY2 import vtkRedistributeDataSetFilter
+    except ModuleNotFoundError:  # pragma: no cover
+        pass  # unavailable on macOS M1 VTK 9.2.0rc1
```

Comment:
I think this is not specifically a MacOS M1 thing, but rather an issue with the VTK conda package. Minor detail, but the comment could lead to some confusion in the future. @larsoner did you try this import with a pip-installed VTK 9.2.0 wheel on your M1 machine?

Reply:
We also covered in slack that it's missing for 9.0 and 9.1 conda-forge VTK.

Reply:
For reference: https://pyvista.slack.com/archives/CBA5095UH/p1661028781796529

Reply:
> I didn't notice that auto-merge was enabled...

My bad. Force of habit, been merging a ton elsewhere. @whophil, can you open up a follow-up?